core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -42,6 +42,17 @@ package object config {
   private[spark] val SPARK_TASK_PREFIX = "spark.task"
   private[spark] val LISTENER_BUS_EVENT_QUEUE_PREFIX = "spark.scheduler.listenerbus.eventqueue"
 
+  private[spark] val DEFAULT_PARALLELISM =
```

Comment:
Can we replace all hardcoded ones with this？

Reply:
+1

Reply:
done